external/mve11/mve_run.m

```diff
@@ -1,286 +0,0 @@
-function [x,E] = mve_run(A,b,x0)
```

Comment:
There was no deeper reason in deleting it, and it could be good to keep the original version of the code there. Maybe we could modify the file name to be mve_run_original or something?